pandas/sparse/series.py

```diff
@@ -570,27 +570,13 @@ def copy(self, deep=True):
         return self._constructor(new_data, sparse_index=self.sp_index,
                                  fill_value=self.fill_value).__finalize__(self)
 
+    @Appender(generic._shared_docs['reindex'] % _shared_doc_kwargs)
     def reindex(self, index=None, method=None, copy=True, limit=None,
                 **kwargs):
-        """
-        Conform SparseSeries to new Index
-
-        See Series.reindex docstring for general behavior
 
-        Returns
-        -------
-        reindexed : SparseSeries
-        """
```

Comment:
So just to be sure I understand: by just removing some obsolete and buggy code, I fix multiple bugs (ignoring a documented argument is a bug too) and leave untested only an argument of which (even in the ``Series`` case) I can't even find a working example, let alone an unambiguous documentation. If you don't even have time to at least be more specific in your replies to my comments... well I don't have time neither.